src/assets/styles/navigation.css

```diff
@@ -0,0 +1,34 @@
+.navigation {
+  background-color: white;
+}
+
+.navContent {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+}
+
+.navContent > li {
+  margin-left: 16px;
+}
+
+.navContent > li > a {
+  text-decoration: none;
+  color: grey;
+}
+
+.navContent > li:last-of-type {
+  margin-left: auto;
+}
+
+.navBar {
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+  margin-left: 40px;
+}
+
```

Comment:
- [ ] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![scrnli_10_3_2021_7-43-04 AM](https://user-images.githubusercontent.com/45217364/135741707-15fa5f5f-4219-4112-870f-46ec359fe8f3.png) |
   - The profile icon must be the icon not the text
   - The color of your progress bar doesn't match the one in the mockup design. you can use the colorPicker tool to pick up the exact color from the mockup.
   - There must not be any text inside the progress bar
   - The app should not be full screen, it must be like a card in the center of the page just like in the mockup design